docs/development/extensions-contrib/k8s-jobs.md

```diff
@@ -284,6 +284,8 @@ To do this, set the following property.
 
 |Property| Possible Values |Description|Default|required|
 |--------|-----------------|-----------|-------|--------|
-|`druid.indexer.runner.k8sAndWorker.workerTaskRunnerType`|`String`|Determines whether the `httpRemote` or the `remote` task runner should be used in addition to the Kubernetes task runner.|`httpRemote`|No|
-|`druid.indexer.runner.k8sAndWorker.sendAllTasksToWorkerTaskRunner`|`boolean`| Whether to send all the tasks to the worker task runner. If this is set to false all tasks will be sent to Kubernetes|`false`|No|
+|`druid.indexer.runner.k8sAndWorker.runnerStrategy.type`|`String`(e.g., `k8s`, `worker`, `taskSpec`)| Defines the task runner type to execute tasks.|`k8s`|No|
+|`druid.indexer.runner.k8sAndWorker.runnerStrategy.workerType`|`String`(e.g., `httpRemote`, `remote`)| Specifies the variant of the worker task runner to be utilized, applicable when the `runnerStrategy.type` is set to `worker` or `taskSpec`.|`httpRemote`|No|
```

Comment:
```suggestion
|`druid.indexer.runner.k8sAndWorker.runnerStrategy.worker.workerType`|`String`(e.g., `httpRemote`, `remote`)| Specifies the variant of the worker task runner to be utilized, applicable when the `runnerStrategy.type` is set to `worker` or `taskSpec`.|`httpRemote`|No|
```

Since workerType only applies to the worker runnerStrategy, should the config be scoped to that strategy?.

I think some example configs of how an operator would set the mixed runner mode would be helpful in evaluating whether this is a naming scheme we want to stick with.

@georgew5656 Do you have thoughts on how these properties should be named?